internal/provider/kubernetes/controller.go

```diff
@@ -226,18 +239,6 @@ func (r *gatewayAPIReconciler) Reconcile(ctx context.Context, _ reconcile.Reques
 			gwcResource.Namespaces = append(gwcResource.Namespaces, namespace)
 		}
 
-		// Process the parametersRef of the accepted GatewayClass.
```

Comment:
curious why we moved this away from L241

Reply:
need update `allAssociatedBackendRefs` first.